diagnostics-app/src/app/diagnostics/diagnostics-card/diagnostics-card.component.html

```diff
@@ -0,0 +1,55 @@
+<!-- 
+ Copyright 2023 The Chromium Authors. All rights reserved.
+ Use of this source code is governed by a BSD-style license that can be
+ found in the LICENSE file.
+-->
+
+<div class="diagnostics-card">
+  <div class="header-bar">
+    <div class="header-and-params">
+      <header>{{ routine.toUpperCase() }}</header>
+      <div class="params-info" *ngIf="params">
+        <div class="param-info" *ngFor="let param of $any(params) | keyvalue: originalOrder">
+          {{ param.key }}: {{ param.value }}
+        </div>
+      </div>
+    </div>
+    <div class="buttons">
+      <button mat-raised-button color="basic" (click)="startRoutine()" [disabled]="state !== 'ready'">Run</button>
```

Comment:
Hi Jessica,

Can you help me add a comment like:

```
TODO(b/301888904): Change comparison to use enum after migration from const enum to enum
```

This will make it easier for me to track where changes need to be make.

Feel free to add it in the top of this file, since there are multiple occurence in this html file.

Reply:
No problem. Done.